modules/standard/Map.chpl

```diff
@@ -103,10 +103,19 @@ module Map {
       :arg valType: The type of the values of this map.
       :arg parSafe: If `true`, this map will use parallel safe operations.
     */
-    proc init(type keyType, type valType, param parSafe=false) {
+    proc init(type keyType, type valType, param parSafe=false, chpl_nilable=true) {
```

Comment:
Do we have to modify the user facing API? Would it be possible to avoid introducing `chpl_nilable` at all if we just made use of `isNonNilableClass`?

Reply:
No. I'll switch it to just call `isNonNilableClass` wherever it was using `chpl_nilable`.